documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc

```diff
@@ -0,0 +1,31 @@
+[id='cli_start-{context}']
+= Connecting to {brandname} Servers running in a container
+Establish CLI connections to {brandname} when running in a container.
```

Comment:
I suggest either:

You can establish a connection with a {brandname} server running in a container through the command line interface (CLI) by following this procedure.

or

You can access the command line interface (CLI) of a {brandname} server running in a container by following this procedure.